modules/schema/pages/codelists-and-fields.adoc

```diff
@@ -29,30 +29,30 @@ the contracting authority shall also consider local legislation.
 [width="100%",cols="<.^40%,<.^60%",options="header",]
 |===
 ^|*Code* ^|*Value* 
-|*CRITERION.EXCLUSION.CONVICTIONS.PARTICIPATION_IN_CRIMINAL_ORGANISATION* |Participation in a criminal organisation
-|*CRITERION.EXCLUSION.CONVICTIONS.CORRUPTION* |Corruption
-|*CRITERION.EXCLUSION.CONVICTIONS.FRAUD* |Fraud
-|*CRITERION.EXCLUSION.CONVICTIONS.TERRORIST_OFFENCES* |Terrorist offences or offences linked to terrorist activities
-|*CRITERION.EXCLUSION.CONVICTIONS.MONEY_LAUNDERING* |Money laundering or terrorist financing
-|*CRITERION.EXCLUSION.CONVICTIONS.CHILD_LABOUR-HUMAN_TRAFFICKING* |Child labour and other forms of trafficking in human beings
-|*CRITERION.EXCLUSION.CONTRIBUTIONS.PAYMENT_OF_TAXES* |Payment of taxes
-|*CRITERION.EXCLUSION.CONTRIBUTIONS.PAYMENT_OF_SOCIAL_SECURITY* |Payment of social security contributions
-|*CRITERION.EXCLUSION.SOCIAL.ENVIRONMENTAL_LAW* |Breaching of obligations in the fields of environmental law
-|*CRITERION.EXCLUSION.SOCIAL.SOCIAL_LAW* |Breaching of obligations in the fields of social law
-|*CRITERION.EXCLUSION.SOCIAL.LABOUR_LAW* |Breaching of obligations in the fields of labour law
-|CRITERION.EXCLUSION.BUSINESS.BANKRUPTCY |Bankruptcy
-|CRITERION.EXCLUSION.BUSINESS.INSOLVENCY |Insolvency
-|CRITERION.EXCLUSION.BUSINESS.CREDITORS_ARRANGEMENT |Arrangement with creditors
-|CRITERION.EXCLUSION.BUSINESS.BANKRUPTCY_ANALOGOUS |Analogous situation like bankruptcy under national law
-|CRITERION.EXCLUSION.BUSINESS.LIQUIDATOR_ADMINISTERED |Assets being administered by liquidator
-|CRITERION.EXCLUSION.BUSINESS.ACTIVITIES_SUSPENDED |Business activities are suspended
-|CRITERION.EXCLUSION.MISCONDUCT.MC_PROFESSIONAL |Guilty of grave professional misconduct
-|CRITERION.EXCLUSION.MISCONDUCT.MARKET_DISTORTION |Agreements with other economic operators aimed at distorting competition
-|CRITERION.EXCLUSION.CONFLICT_OF_INTEREST.PROCEDURE_PARTICIPATION |Conflict of interest due to its participation in the procurement procedure
-|CRITERION.EXCLUSION.CONFLICT_OF_INTEREST.PROCEDURE_PREPARATION |Direct or indirect involvement in the preparation of this procurement procedure
-|CRITERION.EXCLUSION.CONFLICT_OF_INTEREST.EARLY_TERMINATION |Early termination, damages or other comparable sanctions
-|CRITERION.EXCLUSION.CONFLICT_OF_INTEREST.MISINTERPRETATION |Guilty of misinterpretation, withheld information, unable to provide required documents and obtained confidential information of this procedure
-|CRITERION.EXCLUSION.NATIONAL.OTHER |Purely national exclusion grounds
+|*crime-org* |Participation in a criminal organisation
```

Comment:
As the "exclusion grounds" codelist is now a tailored codelist based on the "criterion" published in EU Vocabularies, so the first part of this page should also be reviewed.

I don't think we should indicate in our docs which of the exclusion grounds are recommended by EU legislation. Linking to the legislation is enough.

In consequence, there is no need to duplicate the content of the codelist here, it's better to link to the corresponding page, which is generated automatically (so always up-to-date):
https://docs.ted.europa.eu/eforms/1.3.1/reference/code-lists/exclusion-ground.html
(Link should be defined with the corresponding AsciiDoc syntax)

Reply:
Simplified content. please see latest commit

Reply:
Looks good, thanks!